php/pages/challengeEdit.php

```diff
@@ -1528,12 +1528,13 @@ function getItemOptions() {
 		return ["fauxobjet","banane","carapace","bananeX3","carapacerouge","champi","carapaceX3","poison","bobomb","bloops","pow","champiX3","carapacerougeX3","megachampi","etoile","champior","carapacebleue","billball","eclair"];
 }
 function getItemHitOptions() {
+	/*           fakeItem,   banana,    shell ,     red-shell , bobomb , pow ,  blue shell   , megamushroom, star */
 	var res = ["fauxobjet","banane","carapace","carapacerouge","bobomb","pow","carapacebleue","megachampi","etoile"];
```

Comment:
We probably need to rename the items here too, otherwise the selected distribution won't work correctly